docs/documentation/columns/sizes.html

```diff
@@ -306,7 +307,44 @@
   <p class="message-header">Naming convention</p>
   <p class="message-body">Each modifier class is named after <strong>how many columns you want out of 12</strong>. So if you want 7 columns out of 12, use <code>is-7</code>.</p>
 </div>
-
+<div class="columns">
+  <div class="column is-1">
+    <p class="bd-notification is-primary"><code>is-1</code></p>
+  </div>
+  <div class="column">
+    <p class="bd-notification is-fojeisj has-text-centered">1</p>
+  </div>
+  <div class="column">
+    <p class="bd-notification is-fojeisj has-text-centered">1</p>
+  </div>
+  <div class="column">
+    <p class="bd-notification is-wajofij has-text-centered">1</p>
+  </div>
+  <div class="column">
+    <p class="bd-notification is-fojeisj has-text-centered">1</p>
+  </div>
+  <div class="column">
+    <p class="bd-notification is-wajofij has-text-centered">1</p>
+  </div>
+  <div class="column">
+    <p class="bd-notification is-fojeisj has-text-centered">1</p>
```

Comment:
Can you remove all `is-fojeisj` and `is-wajofij` classes?

Reply:
Arh yeah of course! Should be fixed now. Is that jekyll making those?